### PR TITLE
Fix ecs service creation

### PR DESCRIPTION
**Pull request by mikhail**
@swaggy @diranged

### CODE REVIEW

File: kingpin/actors/aws/ecs.py [hunk 1]
```diff
@@ -917,8 +925,16 @@ def _ensure_service(self, service_name, task_definition_name):
         repeating_log = utils.create_repeating_log(
             self.log.info,
             'Waiting for primary deployment to be updated...', seconds=30)
+
+        while True:
+            try:
+                service = yield self._describe_service(service_name)
+            except ServiceNotFound:
+                yield gen.sleep(2)
```
Comment (diranged) [inline kingpin/actors/aws/ecs.py]: Don't we have a nifty logging tool here to help the user out? :)

Comment (mikhail) [inline kingpin/actors/aws/ecs.py]: Do you mean the logging tool used 3 lines earlier?

**Comment by mikhail** [comment]
Ready to merge?

**Comment by swaggy** [comment]
LGTM.

**Pull request merged**